css/design_system/content.scss

```
@@ -0,0 +1,22 @@
+/**
+ * @title Savile Content
+ * @category content
+ * @description Our base content styles. If this class is added to the body tag (or any container tag), all it's children will receive this styling.
```

I'm sorry. I'm the worst.

```suggestion
 * @description Our base content styles. If this class is added to the body tag (or any container tag), all its children will receive this styling.
```